http4s/src/main/scala/com/avast/grpc/jsonbrige/http4s/Http4s.scala

```diff
@@ -52,26 +54,47 @@ object Http4s extends StrictLogging {
                 request
                   .as[String]
                   .flatMap { body =>
-                    bridge.invoke(GrpcMethodName(serviceName, methodName), body, mapHeaders(request.headers))
-                  }
-                  .flatMap {
-                    case Right(resp) => Ok(resp, `Content-Type`(MediaType.application.json))
-                    case Left(st) => mapStatus(st, configuration)
+                    val methodName0 = GrpcMethodName(serviceName, methodName)
```

Comment:
`0` suffix of variable name here and line below - probably result of an automatic refactoring.

Reply:
that's intentional -- I can't reuse (shadow) the variable `methodName`
open to suggestions :)

Reply:
E.g. rename `methodName` to `methodNameAsString` or something like this - it's IMHO better than `0` suffix :)